docs/wcs/index.rst

```diff
@@ -138,8 +154,8 @@ Supported projections
 
 As `astropy.wcs` is based on `wcslib`_, it supports the standard
 projections defined in the WCS papers.  These projection codes are
-specified in the second part of the ``CUNITn`` keywords (accessible
-through `Wcsprm.cunit <astropy.wcs.Wcsprm.cunit>`), for example,
+specified in the second part of the ``CTYPEn`` keywords (accessible
+through `Wcsprm.ctype <astropy.wcs.Wcsprm.ctype>`), for example,
 ``RA-TAN-SIP``.  The supported projection codes are:
```

Comment:
The SIP suffix is still `astropy.wcs` only.  But I think it's useful as an example to show that a third section is possible.  You're right about the extra hyphens, of course.